argo-cd-apps/base/member/infra-deployments/knative-eventing/knative-eventing.yaml

```diff
@@ -16,16 +16,20 @@ spec:
                 clusterDir: base
           - list:
               elements:
-                - nameNormalized: kflux-ocp-p01
-                  values.clusterDir: kflux-ocp-p01
                 - nameNormalized: stone-stage-p01
                   values.clusterDir: stone-stage-p01
-                - nameNormalized: stone-stg-rh01
-                  values.clusterDir: stone-stg-rh01
```

Comment:
it seems to me this should not be removed

Reply:
Right, added back.